war/.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
```

Comment:
I've done a quick test and it looks like not required and it's preferable not to use this, but then we need to decide if we are using the zero-install mode or not.

It appears you've setup the git ignore file for the zero-install mode but it's not working because there's a `*.zip` ignore rule somewhere preventing it working.

if we got the zero install mode then we will commit 34mb of zip files containing dependencies.

and a 1.1mb `.pnp.cjs` file.

We would also need to add a `yarn install --check-cache` to CI to make sure no one tries to inject a dodgy dep.

We would also need to disable the `yarn install` in the pom file as it wouldn't be needed anymore.

Alternatively we can not use the zero install mode and then we should just add `.pnp.*` to the git ignore file.

Any thoughts?

Reply:
any opinion @StefanSpieker @fqueiruga @janfaracik ?

Reply:
> do we need node_modules?

I tried removing it but then `yarn build` just hung. I don't think this needs to be tackled right away. I suspect something in the ancient dependency tree in package.json is tripping it up.

Reply:
Seems to work fine on my machine, I'll file a PR after this is merged